### ISSUE TITLE
Billing out of Shopify Singapore

### ISSUE THREAD

**Issue by anyacasey** [sharpaypal/financial-services-support#46]
## Product Launch or Change 

**Give a TLDR of what the launch or change is.**

We will be starting to bill merchants based in APAC countries from our new Singapore billing entity (Shopify Singapore Billing). 
As part of this change, we will have to charge GST to all merchants being billed out of Shopify Singapore. Merchants will not be asked to provide a tax number for Singapore, and they will not have an option to declare themselves tax-exempt.

Prior to this change, we have not been collecting GST from merchants in Singapore. Shopify will notify the impacted merchants at least 30 days before the change takes effect.

**Link relevant project resources**

- [Project board](https://vault.shopify.io/projects/8893)

- [Care package](https://docs.google.com/document/d/1NFQBop7zBS4wqKtu4WyJO5Qd9qmEQghM6-mAeNE9M6o/edit)

**From what you know, what tasks do we need to complete to be ready?**

- Meet with PSN to gather context/ gauge scope 
- Review care package
- Arrange team access to Stripe SG
- Work with PSN to create Guru card 
- Create team Trello & update FSS

**What is the estimated launch date?**

Initial build should be completed by mid-October

@anyacasey @deborahisaac @sharpaypal

**Comment by anyacasey** [comment]
**October 1st Meeting with PSN**

* This is part of Shopify’s move away from billing out of Shopify Inc (Canada), similar to the launch of Shopify International, to bill out of different global entities

* **Default to closed:** it’s important that we don’t explicitly state in any of our comms that Shopify is setting up a separate billing entity in Singapore 

* TL;DR: we are starting to charge GST on invoices in Singapore. Merchants will not have the option to declare themselves tax-exempt in Shopify, and they will not be asked to provide a tax number. If they are GST exempt, they can submit a request to their local tax authority at the end of the tax year to reclaim taxes paid

* PSN will be working closely with the legal and tax teams on comms

* When it launches, merchant’s will see GST charged on their **next bill** (no prorated charges)

* Launch is on track for mid/end October [pre-BFCM]

* The Unicorn board is the best place to check for updates: https://vault.shopify.io/projects/8893

**Comment by anyacasey** [comment]
**Rollout Plan** can be viewed [here](https://docs.google.com/document/d/1oQMtCNzWZpPmeEwQ-4K8eU1tBADWfuHB5MmRfMxXSbY/edit)

**Comment by anyacasey** [comment]
**October 22nd Meeting with PSN:**

* Guru card content created & ready to go, pending some details
* The GST will not be applied to Shopify Payments fees
* Bill display: we've uncovered an error in how merchants downloaded bills for Ireland do not show the pre $ conversion VAT amount in Euro; this is something that will need to be fixed ahead of launch (as the Singapore bill breakdown will follow this format)

* #billing-singapore for updates

**Comment by anyacasey** [comment]
**October 30th Update:**

* Guru card published [here](https://app.getguru.com/card/TpkEka8c/Billing-Sales-Tax-Singapore)
* Team Trello update shipped [here](https://trello.com/c/wz4thI0g)
* #billing-singapore for further updates
